your_first_robot/part1/post-es.md

```diff
@@ -0,0 +1,82 @@
+
+Bueno, sigamos la [guía de instalación de Raspberry Pi 2 o 3](https://developer.ubuntu.com/core/get-started/raspberry-pi-2-3) (comente aquí si tiene alguna pregunta). Al final de este paso, usted tendrá Ubuntu Core instalado, y debería poder entrar al Pi a través de SSH.
+
+## Step 2: Preparar el ambiente de desarrollo
+
+Como un sistema operativo enfocado en IoT, Ubuntu Core usa un formato de empaquetamiento diferente al de Raspbian y Ubuntu clásico. En lugar de usar paquetes Debian (que involucran familiaridad con herramientas como **apt**), Ubuntu Core usa un formato de paquetes llamado **snaps**. Los snaps son mucho más robustos cuando se trata de actualizaciones y seguridad, ambas importantes para dispositivos IoT, incluyendo robots. No hay **apt** en este sistema. Para darle una prueba de cómo es usar los comandos «snap», asegúrese que su sistema está completamente actualizado corriendno **snap refresh**.
+
+Entonces, ¿cómo instala una todas las herramientas de desarrollo a las que está acostumbrada? Hay un snap especial para eso, llamado **classic**, que le da acceso a todas su herramientas conocidas (incluyendo **apt**).
+
+Instale el snap **classic** siguiendo la sección de «developing on target» (desarrollando en el mismo sistema objetivo) de la [guía de configuración de desarrollo](https://developer.ubuntu.com/core/get-started/developer-setup) (de nuevo, comente aquí si tiene preguntas). Al final de este paso, usted debe poder ejecutar **sudo classic** y obtener acceso a un shel en el que puede instalar debs, ¡el cual usaremos para hackear en nuestro robot! Luego crearemo su propio snap para controlar el robot, lo que hace super fácil.
+
+## Step 3: Hora de CamJam
+
+Muy bien, ¡ahora estamos en el punto en el que podemos empezar a seguir las hojas de trabajo de CamJam! ¿Por qué no empezamos con la [primera](https://github.com/CamJam-EduKit/EduKit3/raw/master/CamJam%20EduKit%203%20-%20Robotics%20Worksheet%201%20-%20Introduction.pdf). Esta en realidad asume que usted está ejecutando Raspbian, lo que no estamos haciendo, entonces aunque puede leerla toda, vamos a saltar directo al paso «Identifying your Version of Raspbian» (Identificando su versión de Raspbian). No, aún no estamos usando Raspbian, pero esta sección tiene un punto interesante que quiero asegurarme de que note:
+
+Ubuntu Core, al igual que Debian Wheezy, considera que el acceso a GPIO a través de mapeo de memoria es una operación privilegiada. Como resultado, cualquier código a lo largo de estas hojas de trabajo que usa GPIO necesitará que lo ejecute con **sudo**.
+
+Todo bien, empecemos. Asegúrese que está en su shell classic ejecutando **sudo classic**. vi está disponible aquí, pero puede **sudo apt install nano** si quiere. Siguiendo las hojas de trabajo, abra **1-helloworld.py** y llénelo con lo siguiente:
```

Comment:
vi está disponible aquí > vi está disponible predeterminadamente aquí

Reply:
We don't need to make a literal translation, and "predeterminadamente" sounded bad to me. Do you think it makes it better?